libs/search-widget/src/widgets/search-video-widget/SearchBar.svelte

```diff
@@ -104,6 +105,7 @@
     setLang(lang);
 
     setupTriggerSearch(dispatchCustomEvent);
+    loadFieldData();
```

Comment:
loading field data is needed on the results, not really on the SearchBar, why not calling this method in SearchResults?